PolyEngine/Engine/Src/Engine.hpp

```diff
@@ -11,9 +11,8 @@
 
 #include "InputSystem.hpp"
 
-namespace Poly 
+namespace Poly
 {
-	class World;
```

Comment:
How this file compile if `class World` was removed? This is undefined symbol now here `World& GetWorld() { return *BaseWorld; }` and here `World* BaseWorld;`

Don't know if this break linux build but it definetly breaks Windows one.